tests/lib/ignored-paths.js

```diff
@@ -164,6 +164,30 @@ describe("IgnoredPaths", () => {
                 )
             );
         });
+
+
+    });
+
+    describe("caching file reads", () => {
+
+        let readFileSyncCount;
+
+        before(() => {
+            readFileSyncCount = sinon.spy(fs, "readFileSync");
```

Comment:
there isn't a sinon sandbox used elsewhere in the file, and just to double check, are you referring to sandbox as in http://sinonjs.org/releases/v1.17.7/sandbox/ ? @platinumazure